README.md

```diff
@@ -41,8 +41,8 @@ var faceInformation = new ConnectionInformation(baseUrl, subscriptionKey);
 var faceClient = new FaceClient(faceInformation);
 
 // Verifies the faces similarity between two images in base 64
-VerifyImagesResponse verifyImages = await faceClient.VerifyImagesAsync(firstPhotoInBase64, secondPhotoInBase64);
-Console.WriteLine($"Similarity Score: { verifyImages.Score }");
+MatchingScore imagesMatchingScore = await faceClient.VerifyImagesAsync(firstPhotoInBase64, secondPhotoInBase64);
```

Comment:
This should be MatchingResponse, right? I think this MatchingResponse class was not commited. Please check.